saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLSensors.scala

```diff
@@ -69,25 +69,42 @@ object MultiModalSpRLSensors {
     "child-boy" -> "child",
     "child-girl" -> "child",
     "construction-other" -> "construction",
-    "couple-of-persons" -> "persons",
+    "couple-of-persons" -> "humans",
     "face-of-person" -> "person",
+    "group-of-persons" -> "humans",
+    "hand-of-person" -> "person",
+    "head-of-person" -> "person",
+    "person-related-objects" -> "person",
     "floor-other" -> "floor",
     "floor-wood" -> "floor",
-    "group-of-persons" -> "persons",
-    "hand-of-person" -> "hand",
-    "kitchen-pot" -> "pot",
+    "floor-carpet" -> "floor",
+    "floor-tennis-court" -> "floor",
+    "branch" -> "trees",
     "non-wooden-furniture" -> "furniture",
-    "plant-pot" -> "pot",
-    "public-sign" -> "sign",
-    "ruin-archeological" -> "ruin",
+    "wooden-furniture" -> "furniture",
+    "furniture-other" -> "furniture",
+    "public-sign" -> "public-sign",
+    "ruin-archeological" -> "edifice",
     "sand-beach" -> "beach",
     "sand-dessert" -> "dessert",
     "sky-blue" -> "sky",
     "sky-light" -> "sky",
     "sky-night" -> "sky",
     "sky-red-sunset-dusk" -> "sky",
     "water-reflection" -> "water",
-    "wooden-furniture" -> "furniture"
+    "air-vehicles" -> "vehicle",
+    "ground-vehicles" -> "vehicle",
+    "vehicles-with-tires" -> "vehicle",
+    "public-sign" -> "sign",
+    "ancent-building" -> "edifice",
+    "boat-rafting" -> "vehicle",
+    "polar-bear" -> "bear",
+    "fowl-hen" -> "bird",
+    "flock-of-birds" -> "animal",
+    "school-of-fishes" -> "animal",
+    "man-made-other" -> "man-made",
+    "ocean-animal" -> "animal",
+    "herd-of-mammals" -> "animal",
+    "mammal-other" -> "mammal"
   )
```

Comment:
the PR is ok to merge, but @Rahgooy  please check the `phraseConceptToWord = HashMap(...`